pycfmodel/action_expander.py

```diff
@@ -34,9 +34,9 @@ def _expand_actions(actions: Union[str, List[str]], not_action=False) -> List[st
 def expand_actions(obj):
     if isinstance(obj, dict):
         for key, value in obj.items():
-            if key == "Action":
+            if key == "Action" and value is not None:
```

Comment:
```suggestion
            if value is None:
                continue # or obj[key] = None
            elif key == "Action" and value:
```
Otherwise we'll end up recursively calling expand actions when we probably should not

Reply:
Done